app/scripts/views/sign_up.js

```diff
@@ -257,6 +258,15 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       this.navigate('cannot_create_account');
     },
 
+    _isSyncChooser: function () {
+      var chooseSyncCapability = this.broker.getCapabilityByName('choose_what_to_sync');
+      // if this is sync and broker is capable of using choose what to sync on the web v1
+      if (this.relier.isSync() && chooseSyncCapability && chooseSyncCapability.version === 'web_v1') {
```

Comment:
The more I look at this PR, the more I think the broker should make this decision. You could do something like:

```js
  if(this.broker.supportsChooseSyncCapability('web_v1')) {
    ...
```

I'd also move that `web_v1` into a constant somewhere, either in constants.js or onto the base broker.

It almost seems like a separate capabilities model is needed, but maybe not yet.

Are all capabilities versioned? How will this scheme work? Is there any other data associated with a capability? Could the name be simplified to `choose_what_to_sync_web_v1`? If there is a new version, it could be `choose_what_to_sync_web_v2` and so on?